tests/test_preprints.py

```diff
@@ -306,6 +311,775 @@ def test_withdraw_preprint(self, session, driver, preprint_detail_page):
             )
 
 
+@markers.dont_run_on_prod
+class TestPreprintModeration:
+    def test_accept_pre_moderated_preprint(self, session, driver, must_be_logged_in):
+        """Test the acceptance of a preprint submission to a Preprint Provider with a
+        Pre-moderation workflow. In this workflow a preprint is submitted to the
+        preprint service provider but is not yet published. A moderator must then
+        'accept' the preprint submission before it is published and publicly accessible.
+        NOTE: In this test case User One is used to login to OSF and must therefore be
+        setup through the admin app as a moderator or admin for the Preprint Provider
+        being used.  The test will use the OSF api to create a submitted preprint, but
+        the preprint will be submitted using the session credentials of User Two so
+        that the preprint submitter and moderator are different users.
+        """
+        # The following Preprint Provider must be setup in each testing environment.
+        provider_id = 'selpremod'
+        provider_name = 'Selenium Pre-moderation'
+        preprint_title = 'OSF Selenium Pre-moderation Preprint'
+
+        # NOTE: Using User Two to create the preprint through the api so that the user
+        # that submits the preprint is different from the user that accepts or rejects
+        # it.
+        session_user_two = osf_api.get_user_two_session()
+        preprint_node = osf_api.create_preprint(
+            session_user_two,
+            provider_id=provider_id,
+            title=preprint_title,
+            license_name='CC0 1.0 Universal',
+            subject_name='Engineering',
+        )
+        # Use the api to verify that the Preprint is not yet published and that its
+        # review status is 'pending'.
+        assert not (
+            osf_api.get_preprint_publish_and_review_states(preprint_node=preprint_node)[
```

Comment:
you call `osf_api.get_preprint_publish_and_review_states(preprint_node=preprint_node)` twice, once here then again a few lines below.  Could you save its retval into an temp variable and assert on that instead?  It would also hopefully fix `black`'s dreadful indentation of the array index.

Reply:
Good call. The black indentation is really ugly for these lines.

Reply:
Fixed.